stock_valued_picking_report/models/stock_pack_operation.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014 Pedro M. Baeza - Tecnativa <pedro.baeza@tecnativa.com>
+# Copyright 2015 Antonio Espinosa - Tecnativa <antonio.espinosa@tecnativa.com>
+# Copyright 2016 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, fields, models
+from openerp.addons.decimal_precision import decimal_precision as dp
+
+
+class StockPackOperation(models.Model):
+    _inherit = "stock.pack.operation"
+
+    currency_id = fields.Many2one(
+        related='sale_line.currency_id',
+        string='Currency')
+    sale_line = fields.Many2one(
+        comodel_name='sale.order.line',
+        compute='_compute_sale_order_line_fields',
+        string="Related order line")
+    sale_tax_id = fields.Many2many(
+        comodel_name='account.tax',
+        compute='_compute_sale_order_line_fields',
+        string="Taxes")
+    sale_tax_description = fields.Char(
+        compute='_compute_sale_order_line_fields',
+        string='Tax Description')
+    sale_price_unit = fields.Float(
+        compute='_compute_sale_order_line_fields',
+        digits=dp.get_precision('Product Price'),
+        string="Sale price unit")
+    sale_discount = fields.Float(
+        compute='_compute_sale_order_line_fields',
+        digits=dp.get_precision('Discount'),
+        string="Sale discount (%)")
+    sale_price_subtotal = fields.Monetary(
+        compute='_compute_sale_order_line_fields',
+        string="Price subtotal")
+    sale_price_tax = fields.Float(
+        compute='_compute_sale_order_line_fields',
+        string='Taxes')
+    sale_price_total = fields.Monetary(
+        compute='_compute_sale_order_line_fields',
+        string='Total')
+
+    @api.multi
+    def _compute_sale_order_line_fields(self):
+        for operation in self:
+            sale_lines = operation.mapped(
+                'linked_move_operation_ids.move_id.procurement_id.'
```

Comment:
@pedrobaeza Added to Known Issues and conditional hide of amounts in the report for such case.